src/dags/common/db.py

```diff
@@ -19,6 +19,12 @@ def __init__(self, postgres_conn_id="postgres_default"):
         self.db = self.connection.schema
 
 
+def get_connection(postgres_conn_id: str = "postgres_default") -> Engine:
+    """Return database connection."""
+    connection = PostgresHook(postgres_conn_id=postgres_conn_id)
```

Comment:
Naming the `PostgresHook` `connection` is kind of confusing. `PostgresHook` is not a connection, it encapsulates one, namely `psycopg2._psycopg.connection.connection` to be retrieved using `PostgresHook`'s `get_conn` method.

What I'm saying that the message that this function conveys is rather ambiguous:
- the function is called `get_connection`
- that does not *seem* to return a connection, but an `Engine`
- though the implementation, although the instance is named `connection`, in reality returns a `PostgresHook` instance.

Reply:
You are right, the get_connection is confusing and does not fit. I will change it's name to be more appropriate